es-app/src/views/gamelist/DetailedGameListView.cpp

```diff
@@ -24,6 +26,21 @@ DetailedGameListView::DetailedGameListView(Window* window, FileData* root) :
 	mList.setAlignment(TextListComponent<FileData*>::ALIGN_LEFT);
 	mList.setCursorChangedCallback([&](const CursorState& /*state*/) { updateInfoPanel(); });
 
+	// Thumbnail
+	mThumbnail.setOrigin(0.5f, 0.5f);
+	mThumbnail.setPosition(mSize.x() * 0.65f, mSize.y() * 0.08f);
```

Comment:
Nothing meaningful, but from reading the code it's not clear why is the default for the new element not off screen as well?

How will this behave with themes that do not explicitly define the thumbnail element?

Reply:
aah I see my mistake after reading your comment, I've set a default pos so it's ok with the default carbon theme, but I forgot that the default theme do not include the thumbnail element, so it's displayed even if the theme creator do not want it... I was thinking it will be more "convenient" but in fact it's not from a theme creator view.

How do I submit this little change ? Do I have to make a new PR ?

Reply:
You can update the current PR, submit a new commit on top of this one and then squash them to a single commit again. Thanks!

Reply:
Actually, there is a better way to handle this then positioning it offscreen.  #575 added support for setting the visibility of a theme element.  This was used in #623 when adding additional images to the video view.  You can set the new images to not be visible by default.  
```
mThumbnail.setVisible(false);
```

Reply:
@jrassa so do I remove the setPosition and replace it with setVisible, or do you want both ?

Reply:
I would keep both.  Thats what we went with in #623.

Reply:
argh just committed with a replacement haha, how do I squash the commits after my (re)modification ?